<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SPA GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>SIMPLE Source Code Editor</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="CodeEditor" name="txtCodeEditor"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="title">
       <string>PQL Query - input one query at a time</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTextEdit" name="txtQuery">
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="title">
       <string>Query Results</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTextEdit" name="txtResult"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnLoadQuery">
      <property name="text">
       <string>Load SIMPLE source code from file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnRunQuery">
      <property name="text">
       <string>Evaluate PQL qeury</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QTextEdit</extends>
   <header>codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnLoadQuery</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnLoad_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>635</x>
     <y>683</y>
    </hint>
    <hint type="destinationlabel">
     <x>625</x>
     <y>784</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRunQuery</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnRun_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>714</x>
     <y>710</y>
    </hint>
    <hint type="destinationlabel">
     <x>692</x>
     <y>808</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btnLoad_clicked()</slot>
  <slot>btnRun_clicked()</slot>
 </slots>
</ui>
